### PR TITLE
Logic

### PR DESCRIPTION
**Pull request by Rim-777**

### CODE REVIEW

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        with:
+          elixir-version: "1.16" # [Required] Define the Elixir version
+          otp-version: "26.0" # [Required] Define the Erlang/OTP version
+      - name: Restore dependencies cache
+        uses: actions/cache@v3
+        with:
+          path: deps
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+          restore-keys: ${{ runner.os }}-mix-
+      - name: Install dependencies
```
Comment (am-kantox) [inline .github/workflows/elixir.yml:30]:
```suggestion
      - name: Start distribution
        run: epmd --daemon
      - name: Install dependencies
```

File: test/echo_ping_pong_test.exs [hunk 2]
```diff
@@ -1,3 +1,10 @@
 defmodule EchoPingPongTest do
   use ExUnit.Case
+  doctest EchoPingPong
+
+  test "pings a server" do
+    {:ok, _pid} = Node.start(:test@localhost)
```
Comment (am-kantox) [inline test/echo_ping_pong_test.exs:6]: This line looks better in `test_helper.ex` right before `ExUnit.start()`.

**Review by am-kantox — APPROVED** [review]
:zap: :droplet: :zap: :gem: :zap: :blue_heart: :zap:

I liked the research you’ve made to start a node, this is really great. I mean it, nothing is more valuable than thinking beyond frames of known to accomplish the actual problem.

File: .github/workflows/elixir.yml (hunk 1 above)
Comment (Rim-777) [reply .github/workflows/elixir.yml:32]: Helped, thanks a lot! The only option should be -daemon (one dash)

**Pull request merged**